.jenkins/Jenkinsfile

```diff
@@ -57,7 +57,7 @@ def ACCContainerTest(String label) {
     }
 }
 
-def checkPreCommitRequirements() {
+def checkOEInstallationScripts() {
```

Comment:
I disagree with @anitagov here. These two stages don't seem to be checking the "OE Installation Scripts" at all.

I originally wrote two tests here:

1. Run the check-ci script which repeats the pre-commit hook but across the entire repo.

2. Run a build without any environment variables defined. This was a regression test asserting that the build worked with CC and CXX unset, as we'd hit a bug where CI passed but dev machines broke due to this difference.

So these two tests to me are "Check Developer Experience" (I don't know if that's a good name, but that's what they do).

Reply:
Thanks for your clarification @andschwa . We really need this stage especially bullet 2. How about "checkDevFlows" as a name.

Reply:
That is good! Thanks Anita.